.jest/setup-file-after-env.js

```diff
@@ -2,7 +2,12 @@ import '@testing-library/jest-dom'
 import globalJsdom from 'global-jsdom'
 
 const cleanup = globalJsdom()
-beforeEach(cleanup)
+
+beforeEach(() => {
+  cleanup()
+  global.window = global.window || {}
+  global.window.navigator = global.window.navigator || {}
```

Comment:
not entirely sure what was happening here but there are times when the testing library attempts to access properties in navigator when window is undefined. Will try and dig deeper but this covers it for now.